modules/styles/product/pom.xml.versionsBackup

```diff
@@ -0,0 +1,90 @@
+<!--
+ ~ Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
add `pom.xml.versionsBackup` into gitignore file and we should not commit it

Reply:
Done. Will make changes.

Reply:
PR automatically closed with head reset. Changes in new PR https://github.com/wso2/product-is/pull/13704.